dll/win32/shell32/CDefView.cpp

```diff
@@ -1734,11 +1734,12 @@ LRESULT CDefView::OnSize(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandled
 
     TRACE("%p width=%u height=%u\n", this, wWidth, wHeight);
 
+    // WM_SIZE can come before WM_CREATE
+    if (!m_ListView)
```

Comment:
But note that the original code somehow checked as well this pointer before calling the MoveWindow. But it also called the other functions, so I wonder in which of these other functions down below the crash actually happened.

Reply:
`UpdateScrollbar` uses `m_ListView`. This causes assert failure.